### ISSUE TITLE
Making Work Manager work for you

### ISSUE THREAD

**Issue by Ayusch** [grofers/talks#15]
# Title
> Making Work Manager work for you

## Description
> Work Manager recently hit stable release in March and it's a good time to talk about why work manager is the best scheduling API available to android developers right now.

The talk would begin by bringing up all the existing scheduling APIs one by one and shooting them down and bringing up work manager at the end.

The second part would involve developing a basic app with work manager.

The last part would involve an API walkthrough and some nitty-gritties of work manager.

So this would be structured as a 3 part talk.

## Duration
- [x] 30 min
- [ ] 45 min

## Audience
> Intermediate

## Outline
> A detail flow/outline can be found on this article: https://ayusch.com/work-manager-in-android/
Talk would proceed along the lines of the article.

## Additional notes
> None.

---

- [ ] Don't record this talk.
> Check this if you don't want your talk to be recorded.

**Comment by vinayak-mehta** [comment]
@Ayusch Are you available to give this talk on Saturday?

**Comment by Ayusch** [comment]
Sure.
Will Grofers pay for the travel and accommodation? I'm based in Delhi.

**Comment by Ayusch** [comment]
@vinayak-mehta ?

**Comment by vinayak-mehta** [comment]
No, we're not sponsoring travel and accommodation for this meetup.

**Comment by vinayak-mehta** [comment]
@Ayusch You can probably give this talk at a meetup in our Gurgaon office. @kasisnu can help here.

**Comment by Ayusch** [comment]
Sure :)
